.github/workflows/coverage.yml

```diff
@@ -31,6 +31,6 @@ jobs:
         uses: codecov/codecov-action@v4
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          directory: ./coverage
```

Comment:
NOTE: I will eventually use vitest workspace setup too. This will allow all packages to define tests and the workspace to handle coverage as a whole